### DISCUSSION TITLE
Cómo agrego barras de desplazamiento a la CTkTable? How to add scrollbars in CTkTable?

### DISCUSSION THREAD

**Discussion by feerxz** [Akascape/CTkTable#36] [Q&A] [answered]
Hola @Akascape tu tabla me resultó muy útil para un proyecto que estoy realizando, pero quisiera que me ayudes diciéndome cómo puedo agregar barras de desplazamiento horizontal y vertical, por favor, y gracias

**Comment by Akascape** [comment, marked as answer]
@feerxz Hi, for adding vertical/horizontal scrollbars, you can use my [CTkXYFrame](https://github.com/Akascape/CTkXYFrame) widget. One example with CTkTable is given [there](https://github.com/Akascape/CTkXYFrame/blob/main/example_2.py). Hope this will help.

`Spanish Translation`

@feerxz Hola, para agregar barras de desplazamiento verticales/horizontales, puede usar mi widget [CTkXYFrame](https://github.com/Akascape/CTkXYFrame).  Se proporciona un ejemplo con CTkTable [allí](https://github.com/Akascape/CTkXYFrame/blob/main/example_2.py).  Espero que esto ayude 😉

![screenshot](https://github.com/Akascape/CTkXYFrame/assets/89206401/859b6733-3c91-4093-a511-d6f1060a18ca)

**Reply by feerxz** [reply]
Muchas gracias @Akascape , me sirvió de mucho!!
